index.html

```diff
@@ -144,6 +152,75 @@ <h1 class="entry-title">CODESHOWS</h1>
 
 
 
+      <div class="subcontent" id="projects">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+	  <div class="container-fluid">
+	    <button type="button" class="btn btn-info sidebarCollapse">
+	      <i class="fas fa-align-left"></i>
+	    </button>
+	  </div>
+	</nav>	
+	<div class="container projects">
+	  <div class="container1">
+	    <div class="main">
+	      <div class="heading"> PROJECTS ON CODESHOWS </div>
+	      <hr>
+	      <div id="info">
+		<p> This section contains the ongoing and completed projects by the club :</p> 
+	      </div>
+	      <div class="con">
+                <div onclick="showans('ans7')">
+                  <i class="fas fa-book"></i>
+                  PROJECT 1: <p>Codeshows Website : Codeshows website is open for development.
+		  Go through it's code and contribute in it's development.</p>
+                  <i class="fas fa-sort-down" style="float:right"></i>
+                </div>
+                <p class='p-3' id="ans7"><a href="http://codeshows.github.io/index.html#">Codeshows Website
+                  </a>
+                <br>
+	        <a href="http://github.com/Codeshows/codeshows.github.io">Codeshows Github Link
+                  </a>
+                </p>	
+              </div>
+              <div class="con">
+                <div onclick="showans('ans1')">
+                  <i class="fas fa-book"></i>
+                 PROJECT 2: <p>Interview-checklist : A repository for interview related contents. 
+		  Find the relevant content, add it to the repository and eventually benefit from it.</p>
+                  <i class="fas fa-sort-down" style="float:right"></i>
+                </div>
+                <p class='p-3' id="ans1"><a href="http://github.com/Codeshows/interview-checklist">Codeshows/Interview-Checklist
+                  </a>
+                </p>
+              </div>
+ 	      <div class="con">
+                <div onclick="showans('ans2')">
+                  <i class="fas fa-book"></i>
+                 PROJECT 3: <p>Codeshows IDE : The Codeshows-IDE is uploaded on GitHub as a project.
+	          Go through the code and help in it's development.</p>
+                  <i class="fas fa-sort-down" style="float:right"></i>
+                </div>
+                <p class='p-3' id="ans2"><a href="http://github.com/codeshows/codeshows-ide">Codeshows IDE Github Link
+                  </a>        
```

Comment:
Put the description to `p class="p-3"`, the title i.e. `Codeshows IDE` must remain at the current position.
Also, if you can make it a little more attractive, it would be better, maybe use something as used in `Contests` section.
Atleast give a border to distinguish between projects. @Surya1231 can you help her with frontend a bit?

Reply:
Even if you are unable to, it's still fine. I don't expect you to be great at UI/UX yet